src/app/add-new-todo-list/add-new-todo-list.component.html

```diff
@@ -0,0 +1,6 @@
+<h1>{{title}}</h1>
+<input type='text' placeholder='Add new todo item'>
+<div *ngFor='let todo of data.todos'>
+  {{todos.title}}
+  </div>
```

Comment:
This does not **add** a todolist. It visualise it (you should leave this task to @JMstd